### ISSUE TITLE
Record ideas

### ISSUE THREAD

**Issue by evancz** [elm-lang/elm-compiler#1068]
**This is a meta issue only for collecting related things in one place. If you think something is relevant, do not comment here, open a separate issue.**

People often ask about certain things, so I am collecting all the common requests in one place. I don't expect to be making any major changes here right now. This is more for book keeping.

  - ~~#985 - remove extension and deletion~~
  - #842 - allow more JS-ish style for closing }
  - #635 - allow qualified records in update syntax
  - #537 - allow arbitrary expressions in update syntax
  - #1254 - allow arbitrary types in type extension syntax

**Comment by Apanatshka** [comment]
Maybe add #984 to the list, since it's waiting on 0.16 and #985?

**Comment by jvoigtlaender** [comment]
Maybe add https://github.com/elm-lang/elm-compiler/issues/1254 to the list?

**Comment by evancz** [comment]
Moved all this into #1375

**Comment by jvoigtlaender** [comment]
How about @Apanatshka's proposal to add #984 as well? Seems lost in #1375.

**Comment by evancz** [comment]
I considered that and decided against it. I know that proposal. It's a language feature proposal, not a simple syntax thing.
